development/guideline/communication.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Communication guidelines
+tags: [guidelines, development]
+---
+
+# Communication guidelines
+
+To communicate with and reach a wide audience, we use the following channels
+
+- this website, for documentation
+- the email [discussion list](/discussion_list), for interactions between users and support
```

Comment:
perhaps be really explicit that it's not users versus support, but also to promote interactions between users and users...